src/android/java/com/pspdfkit/cordova/action/annotation/AddAnnotationAction.java

```diff
@@ -0,0 +1,35 @@
+package com.pspdfkit.cordova.action.annotation;
+
+import androidx.annotation.NonNull;
+
+import com.pspdfkit.annotations.AnnotationProvider;
+import com.pspdfkit.cordova.CordovaPdfActivity;
+import com.pspdfkit.cordova.PSPDFKitCordovaPlugin;
+import com.pspdfkit.cordova.action.BasicAction;
+import com.pspdfkit.document.PdfDocument;
+
+import org.apache.cordova.CallbackContext;
+import org.json.JSONArray;
+import org.json.JSONException;
+
+public class AddAnnotationAction extends BasicAction {
```

Comment:
A brief description in its Javadoc would increase readability of this class. Something like "Action to import an annotation from Instant JSON." would already be of great help.